### ISSUE TITLE
Tokenization of certain `(de|em)+(PRON|DET)` is not using ranges

### ISSUE THREAD

**Issue by ceramisch** [UniversalDependencies/UD_Portuguese-GSD#14]
Tokens matched by the following pseudo-regex are not properly tokenized.

`( de | em ) + ( isso | aquilo | el[ea]s? | ess[ea]s? | aquel[ea]s? )`

OR

`de + ( aqui | ali )`

They are contractions so they should be represented as ranges, followed by the words composing the contraction, such as:

```
7-8 daqui _
7 de ADP
8 aqui PRON
```

By the way, their POS is currently `X`, this can also be fixed with a proper tokenization.

I do not know how this is dealt with in Bosque, any retokenization should be applied homogeneously to both.

**Comment by martinpopel** [comment]
Yes, it seems I forgot to add these to https://github.com/udapi/udapi-python/blob/master/udapi/block/ud/pt/addmwt.py (and no Portuguese native speaker warned me).
When considering consistency, we should look also on [French](https://github.com/udapi/udapi-python/blob/master/udapi/block/ud/fr/addmwt.py) and [Spanish](https://github.com/udapi/udapi-python/blob/master/udapi/block/ud/es/addmwt.py) (I don't say the guidelines must be as similar as possible, but we should be at least aware of the differences and the reasons behind).

**Comment by arademaker** [comment]
@Cristiananc can you try solve this? Let us talk about.

**Comment by arademaker** [comment]
To solve this issue, I still need to solve issues in the udapi code above for dealing with all cases of PT contractions. See https://pt.wiktionary.org/wiki/Apêndice:Combinações_e_contrações_do_português.

**Comment by arademaker** [comment]
related to #9

**Comment by arademaker** [comment]
@martinpopel reported by email:

I noticed xpos=ADPPRON is a good detector of these contractions (multiword tokens).
I also noticed that not all such tokens are handled by the current code ud.pt.AddMwt:

cat pt_gsd-ud-* | udapy -s ud.pt.AddMwt > fixed.conllu
udapy util.Eval node='if $.xpos=="ADPPRON" and not $.multiword_token: count_$.form.lower()+"|"+$.lemma +=1' end="pp(self.count)" < fixed.conllu

Counter({'comigo|_': 13,
        'nele|_': 7,
        'nela|_': 6,
        'daqueles|_': 5,
        'conosco|_': 5,
        'àqueles|_': 2,
        'consigo|_': 1,
        'daquelas|_': 1,
        'contigo|_': 1,
        'a|_': 1,
        'nessa|_': 1,
        'naquilo|_': 1,
        'daquele|_': 1,
        'àquela|_': 1,
        'naqueles|_': 1,
        'neles|_': 1,
        'nelas|_': 1})

I hope it should be straightforward for you to adapt
https://github.com/udapi/udapi-python/blob/master/udapi/block/ud/pt/addmwt.py
to handle also these cases.
